pkgs/top-level/python-packages.nix

```diff
@@ -4522,6 +4537,39 @@ in modules // {
     };
   };
 
+  eve = buildPythonPackage rec {
+    version = "0.6.1";
+    name = "Eve-${version}";
+
+    src = pkgs.fetchurl {
+      url = "https://pypi.python.org/packages/source/E/Eve/${name}.tar.gz";
+      sha256 = "0wf1x8qixkld6liz5syqi8i9nrfrhq4lpmh0p9cy3jbkhk34km69";
+    };
+
+    propagatedBuildInputs = with self; [
+      cerberus
+      events
+      flask-pymongo
+      flask
+      itsdangerous
+      jinja2
+      markupsafe
+      pymongo_2_9_1
+      simplejson
+      werkzeug
+
+    ];
+
+    # tests call a running mongodb instance
+    doCheck = false;
```

Comment:
Is it (with a reasonable amount of effort) possible to run the tests that do not depend on a mongodb instance?